src/models/inventoryModels/inventoryModel.ts

```
@@ -1114,3 +1115,15 @@
 type InventoryModelType = Model<IInventoryDatabase, {}, InventoryDocumentProps>;
 
 export const Inventory = model<IInventoryDatabase, InventoryModelType>("Inventory", inventorySchema);
+
+// eslint-disable-next-line @typescript-eslint/ban-types
+export type TInventoryDatabaseDocument = Document<unknown, {}, IInventoryDatabase> &
```

Biome (1.9.4)

[error] Don't use '{}' as a type.

Prefer explicitly define the object shape. '{}' means "any non-nullable value".

(lint/complexity/noBannedTypes)